### ISSUE TITLE
Kita tak ada idea untuk sample project

### ISSUE THREAD

**Issue by mrwaim** [mrwaim/CubaCubaAngularJs#1]
Jom discuss nak buat sample project apa - chat app? Online shop? Tempat cari kerja? Video Feed?

**Comment by datomnurdin** [comment]
develop simple CMS guna angularJS dengan nodejs/parse.

**Comment by mrwaim** [comment]
Untuk blog posts and comments?

**Comment by datomnurdin** [comment]
Yes, start small then grow big. Then dari situ boleh come out dengan framework sendiri :)

**Comment by mrwaim** [comment]
Nie nak buat AngularJS 1 or 2? Saya prefer typescript

**Comment by firdausramlan** [comment]
Kalau AngularJS v2 pun menarik. Mostly banyak concept dari v1 dah deprecated dalam v2.

For starting, boleh kita create github organization account so we can have better repo / people management

**Comment by firdausramlan** [comment]
how about https://github.com/organizations/angularjsMY

**Comment by firdausramlan** [comment]
ui bootstrap?

**Comment by ridhwandaud** [comment]
nak join , baru blaja angular tp dari ionic

**Comment by firdausramlan** [comment]
angular ionic sama je dengan web app cuma dia banyak extra directive khas untuk mobile
